src/docfx/lib/log/Telemetry.cs

```diff
@@ -73,6 +78,22 @@ public static void SetRepository(string? repo, string? branch)
             s_telemetryClient.Context.GlobalProperties["Branch"] = s_branch;
         }
 
+        public static void TrackDocfxConfig(string fileName, string docsetName, JObject docfxConfig)
+        {
+            var docfxConfigTelemetryValue = JsonUtility.Serialize(docfxConfig);
+            var hashCode = HashUtility.GetMd5Hash(docfxConfigTelemetryValue);
+            if (docfxConfigTelemetryValue.Length > MaxEventPropertyLength)
+            {
+                docfxConfigTelemetryValue = SerializeConfigObjectWithMaxLength(docfxConfig);
+            }
+
+            var dimensions = new Dictionary<string, string>();
+            dimensions["DocsetName"] = docsetName;
+            dimensions["Config"] = docfxConfigTelemetryValue;
+            dimensions["ContentHash"] = hashCode;
```

Comment:
What is the purpose of this hash?

Reply:
`hashcode` will be used by the Feature Utilization team to detect the config change once we remove some nested object from the too-long config